addons/website/views/snippets/s_mega_menu_odoo_menu.xml

```diff
@@ -60,14 +60,27 @@
         <div class="container-fluid border-top s_mega_menu_odoo_menu_footer">
             <div class="row">
                 <div class="col-12 pt8 pb8">
-                    <nav class="nav justify-content-center">
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 14px;"><i class="fa fa-fw fa-2x fa-github"></i></a>
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 12px;"><i class="fa fa-fw fa-2x fa-youtube-play"></i></a>
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 12px;"><i class="fa fa-fw fa-2x fa-twitter"></i></a>
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 12px;"><i class="fa fa-fw fa-2x fa-linkedin"></i></a>
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 12px;"><i class="fa fa-fw fa-2x fa-instagram"></i></a>
-                        <a href="#" class="mx-3 my-2 text-o-color-5" style="font-size: 12px;"><i class="fa fa-fw fa-2x fa-facebook"></i></a>
-                    </nav>
+                    <div class="s_social_media text-center o_not_editable" data-snippet="s_social_media" data-name="Social Media" contenteditable="false">
```

Comment:
![image](https://user-images.githubusercontent.com/10338094/161976646-302a6568-8dbc-44e8-9e71-98c45fe4d1f9.png)
The layout option does not like your DOM